DEVELOPING.md

```diff
@@ -14,6 +14,21 @@ Recommended:
   - View them with `remake --tasks`
 - [locust](https://docs.locust.io/en/stable/what-is-locust.html) - A performance testing tool, required for load testing
 
+## Installing with Helm
+
+We have an example Helm chart that can be used to configure and run the agent.
+Update [values.yaml](./chart/values.yaml) with your Honeycomb API key then run:
+
+```sh
+helm install -f chart/values.yaml hny-network-agent ./chart
```

Comment:
```suggestion
helm install hny-network-agent ./chart
```

Reply:
The main use-case we can see for a values file would be to pass in the Honeycomb API key and maybe replace the API endpoint events are sent to. If we don't use the `-f` flag here, how would you recommend to do that instead?

Reply:
the way it was written it was passing the chart's default values.yaml, but that is equivalent to not providing a `-f` or `--values` flag.

Reply:
Users should provide their own values.yaml file, or use the `--set` flag, to override the default values.yaml.